src/components/FilterSearch.tsx

```diff
@@ -123,14 +128,33 @@ export function FilterSearch({
   );
 
   useEffect(() => {
+    if (matchingFilters.length > 1 && !onSelect) {
+      console.warn('More than one selected static filter found that matches the filter search fields.'
```

Comment:
nit: can we add to the message that we will use the first selected static filter found?

Reply:
what do you think about adding "Picking one filter to display in the input." instead? because technically, it will only pick the first filter it finds if the `currentFilter` is not selected in state anymore. or does that seem too vague?